src/osdc/Objecter.cc

```diff
@@ -2481,6 +2481,8 @@ int Objecter::_calc_target(op_target_t *t, epoch_t *last_force_resend,  bool any
   vector<int> up, acting;
   osdmap->pg_to_up_acting_osds(pgid, &up, &up_primary,
 			       &acting, &acting_primary);
+  unsigned prev_seed = ceph_stable_mod(pgid.ps(), t->pg_num,
+                                       pg_pool_t::calc_bits_of(t->pg_num));
```

Comment:
we need to convert the return value of _pg_pool_t::calc_bits_of(t->pg_num)_ ?
for ceph_stable_mod(), the third parameter bmask=(2**n)-1

On above function, _pgid.ps()_ & _pg_pool_t::calc_bits_of(t->pg_num)_ will be zero in some situation.
It seems need to calculate again like following:
mask = (1 << [calc_bits]) - 1;

If I get any wrong or misunderstanding, feel free to let me know!

thanks!!
vicente